esp-hal/src/macros.rs

```diff
@@ -134,3 +134,13 @@ macro_rules! if_set {
         $set
     };
 }
+
+/// Macro to ignore tokens.
+///
+/// This is useful when we need existence of a metavariable (to expand a
+/// repetition), but we don't need to use it.
```

Comment:
Worth adding the FIXME comment here as well?

Reply:
In fact one here would be sufficient, but it's fine and I'd not poke at this PR again just for this.